scene/main/viewport.cpp

```diff
@@ -1527,17 +1529,22 @@ void Viewport::_gui_show_tooltip() {
 
 	tooltip_owner->add_child(gui.tooltip_popup);
 
+	Window *window = gui.tooltip_popup->get_parent_visible_window();
```

Comment:
In the case that `gui.tooltip_popup` is embedded, it is possible, that its embedder is not the parent visible window.

Can you please try:
```suggestion
	Window *window = gui.tooltip_popup->get_embedder();
	if (window == null) {
		window = ui.tooltip_popup->get_parent_visible_window();
	}
```

Reply:
Change made - thanks!